matRad/doseCalc/matRad_calcLETmask.m

```diff
@@ -0,0 +1,52 @@
+function [LETmaskDirty,LETmaskClean,mLET] = matRad_calcLETmask(dij)
```

Comment:
Do we actually need the mLET matrix apart from calculating dirtyDose? I don't like that we need to create / store it. As a matrix it is useless, because it does not depend on fluence.